easybuild/easyconfigs/g/GROMACS/GROMACS-5.1.2-CrayGNU-2016.03-cuda-7.0.eb

```diff
@@ -0,0 +1,44 @@
+# Contributed by 
+# Luca Marsella (CSCS)
+# Guilherme Peretti-Pezzi (CSCS)
+#
+# Copyright:: Copyright 2012-2013 University of Luxembourg / LCSB, Cyprus Institute / CaSToRC, Ghent University
+# Authors::   Wiktor Jurkowski <wiktor.jurkowski@uni.lu>, Fotis Georgatos <fotis.georgatos@uni.lu>, \
+#             George Tsouloupas <g.tsouloupas@cyi.ac.cy>, Kenneth Hoste
+# License::   MIT/GPL
+# $Id$
+#
+# This work implements a part of the HPCBIOS project and is a component of the policy:
+# http://hpcbios.readthedocs.org/en/latest/HPCBIOS_2012-93.html
+
+name = 'GROMACS'
+version = "5.1.2"
+cudaversion = '7.0'
```

Comment:
make this  `7.0.28`, and use `cudaversion` (or `cudaver`) below to define the dep